cns/types/types.go

```diff
@@ -234,7 +234,7 @@ func init() {
 
 type CnsQueryVolumeRequestType struct {
 	This   types.ManagedObjectReference `xml:"_this" json:"-"`
-	Filter CnsQueryFilter               `xml:"filter" json:"filter"`
+	Filter BaseCnsQueryFilter           `xml:"filter,typeattr" json:"filter"`
```

Comment:
I looked at the latest vcenter builds, the generated go bindings show this:

```
type CnsQueryVolumeRequestType struct {
	This	types.ManagedObjectReference	`xml:"_this"`
	Filter	CnsQueryFilter	`xml:"filter"`
	Selection	*CnsQuerySelection	`xml:"selection,omitempty"`
}
```

This change is related to 'CnsKubernetesQueryFilter' bindings or a different issue ?